---
title: C28125
ms.date: 11/04/2016
ms.prod: visual-studio-dev15
ms.topic: reference
f1_keywords:
- C28125
helpviewer_keywords:
- C28125
ms.assetid: 3f7b8db4-6a64-4480-919e-3f3ceca7f948
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: 732bcb5829afa0703d52e0f23e5544dec1987b6f
ms.sourcegitcommit: 37fb7075b0a65d2add3b137a5230767aa3266c74
ms.translationtype: MT
ms.contentlocale: fr-FR
ms.lasthandoff: 01/02/2019
ms.locfileid: "53912189"
---
# <a name="c28125"></a>C28125
avertissement C28125 : La fonction doit être appelée à partir de dans un bloc try / except bloc

 Le pilote appelle une fonction qui doit être appelée à partir de dans un bloc try / except bloc, telles que [ProbeForRead](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforread), [ProbeForWrite](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforwrite), [MmProbeAndLockPages](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-mmprobeandlockpages).

## <a name="example"></a>Exemple
 L'exemple de code suivant génère cet avertissement :

```
ProbeForRead(addr, len, 4);
```

 L'exemple de code suivant évite cet avertissement :

```

__try
{
    ProbeForRead(addr, len, 4);
}
__except(EXCEPTION_EXECUTE_HANDLER)
{
    Status = GetExceptionCode();
    ... report error status
}
```